tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_store_utils.ts

```diff
@@ -42,37 +42,67 @@ export function findFileIndex(
   );
 }
 
+/**
+ * Converts the object representation of a stack frame into the array
+ * representation.
+ * @param stackFrame
+ * @returns StackFrameAsArray
+ */
+export function stackFrame2StackFrameAsArray(
+  stackFrame: StackFrame
+): StackFrameAsArray {
+  return [
+    stackFrame.host_name,
+    stackFrame.file_path,
+    stackFrame.lineno,
+    stackFrame.function_name,
+  ];
+}
+
+/**
+ * Convertsthe array representation of a stack frame into the object
+ * representation.
+ * @param array
+ * @returns StackFrame
```

Comment:
These are already strongly typed in the parameter/returnType. Omit them (they aren't valid Closure types either; here and above)?

Reply:
Removed them.